### PR TITLE
Connect backend

### LINKED ISSUE #61: shop list page get backend

### PR DESCRIPTION
**Pull request by glu8637**
Refactor card component into new storeCard component
shops can now get data from backend and display on page

### CODE REVIEW

File: frontend/src/lib/components/ui/storeCard/storeCard.svelte [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<script lang="ts">
+  import { page } from '$app/stores';
+	import * as Card from '$lib/components/ui/card';
+
+  export let name: string = "";
+  export let description: string = "";
+  export let picture: string = "";
+  export let address: string = "";
```
Comment (nonorze-VOXX) [inline frontend/src/lib/components/ui/storeCard/storeCard.svelte]: maybe fill something default to know miss fill

**Review by nonorze-VOXX — CHANGES_REQUESTED** [review]
other part is LGTM

File: frontend/src/lib/components/ui/storeCard/storeCard.svelte (hunk 1 above)
Comment (glu8637) [reply frontend/src/lib/components/ui/storeCard/storeCard.svelte]: fix

**Review by nonorze-VOXX — APPROVED** [review]
LGTM

**Pull request merged**